src/VirtualClient/VirtualClient.Dependencies/CUDAAndNvidiaGPUDriverInstallation.cs

```diff
@@ -332,7 +332,7 @@ private List<string> VersionSpecificInstallationCommands(LinuxDistribution linux
                 case LinuxDistribution.Ubuntu:
                     commands.Add("apt update");
                     commands.Add("apt upgrade -y");
-                    commands.Add($"apt install nvidia-driver-{this.LinuxDriverVersion} nvidia-dkms-{this.LinuxDriverVersion} -y");
+                    commands.Add($"apt install nvidia-driver-{this.LinuxDriverVersion}-server nvidia-dkms-{this.LinuxDriverVersion}-server -y");
```

Comment:
Do you know the difference? is the original a superset of this -server package? I'm worried about if this change to impact any other workloads. Please investigate if -server is sufficient, if you are not sure, let's add both

Reply:
We cannot have more than one version on the OS for each of the driver, fabric manager and dkms. If we have more than one version, there will be a conflict and we will have to format the whole OS disk.

The -server versions are for the datacenter GPUs which we use. They may not be the latest versions for the normal GPUs we have, but they will be the exact versions for the datacenter GPUs to run. So, it is perfect to use the -server version.

Reply:
@psingla1210 could you work with Sasanka to verify this on other workloads like superbench and nvstream?

Reply:
sure, i will do that